CMake/cdat_modules/vistrails_external.cmake

```diff
@@ -26,12 +26,8 @@ if (NOT _branch STREQUAL "${refspec}")
             elseif(_url MATCHES "^.*uvcdat-devel.git")
               set(vistrails_branch uvcdat-next)
             endif()
-        else()
-            message(WARNING "Unable to extract url for '${_remote}' using default VisTrails branch")
         endif()
-     else()
-        message(WARNING "Unable to extract remote for '${_branch}' using default VisTrails branch")
```

Comment:
Why remove that one it was kind of nice to know it couldn't get the desired branch